tests/unittests/GraphOptzTest.cpp

```diff
@@ -726,10 +726,10 @@ TEST_F(GraphOptz, BatchNormAfterConvNotOptimizeForTrain) {
 
   EXPECT_EQ(F_->getNodes().size(), 3);
 
-  ::glow::optimize(F_, CompilationMode::Train);
```

Comment:
This is a training-specific use case so unfortunately I think we need to leave this in and clone the graph manually.